src/main/java/com/billyyccc/mssqlclient/impl/protocol/MessageStatus.java

```diff
@@ -0,0 +1,37 @@
+package com.billyyccc.mssqlclient.impl.protocol;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum MessageStatus {
+
+  NORMAL(0x00),
+  END_OF_MESSAGE(0x01),
+  IGNORE_THIS_EVENT(0x02),
+  RESET_CONNECTION(0x08),
+  RESET_CONNECTION_SKIP_TRAN(0x10);
+
+  private static final Map<Integer, MessageStatus> typeMapping = new HashMap<>();
+
+  static {
+    typeMapping.put(NORMAL.value, NORMAL);
```

Comment:
I think you can safely do each put in the enum constructor instead of using a static initializer block

Reply:
I think it might be a overkill to introduce data structures into these classes, so I remove them in https://github.com/BillyYccc/incubator-vertx-mssql-client/pull/3/commits/163ca22ff8f81ba7eaf182596b3d788b7fcaaddf.